step-parent/pom.xml

```diff
@@ -56,7 +56,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<!-- internal dependencies -->
 		<exense-commons.version>2.0.6</exense-commons.version>
-		<step-framework.version>2.0.10</step-framework.version>
+		<step-framework.version>0.0.0-SNAPSHOT</step-framework.version>
 		<step-grid.version>2023.9.11-64fed6672a3eb976721359ae</step-grid.version>
```

Comment:
Wont build on the Build server. Reference to the correct build version of the framework instead